nbdime/tests/test_server_extension.py

```diff
@@ -73,15 +73,13 @@ def test_git_difftool(git_repo2, server_extension_app):
     # Extract config data
     match = _re_config.search(r.text)
     data = json.loads(match.group(1))
-    assert data.pop("mathjaxUrl", "").endswith("MathJax.js")
     assert data == {
         "base": "git:",
         "baseUrl": "/nbdime",
         "closable": False,
         "remote": "",
         "savable": False,
-        "hideUnchanged": True,
-        "mathjaxConfig": "TeX-AMS-MML_HTMLorMML-full,Safe",
+        "hideUnchanged": True
```

Comment:
In Python it is preferable to leave trailing commas in place, as it reduces diff churn.